### PR TITLE
Enhance issue_review_template.md

### LINKED ISSUE #61: Updates of review issue template with more functionality
**Problem description**

Extending the review issue template with informatiuon on release mgmt action beyond the API review

**Possible evolution**

Rename to "Review actions"

Add a section Release actions with the following as tickboxes

API Release Tracker available
Review comments addressed (by Sub Project)
Release PR approved (on behalf of Release Management)
PR merged (by Sub Project codeowner)
Release created within GitHub (by Sub Project codeowner)
Release Tracker updated (M3 date = creation date of release?)

### PR DESCRIPTION
**Pull request by tanjadegroot**
#### What type of PR is this?

Add one of the following kinds:
* enhancement/feature
* documentation
* subproject management

#### What this PR does / why we need it:

For better release mgmt activity tracking.

#### Which issue(s) this PR fixes:

Fixes #61

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_review_template.md [hunk 1]
```diff
@@ -10,18 +10,29 @@ assignees: ''
...
+
+Assign this issue to yourself or another RM team member and follow the below list. 
+When done, tick the box in this issue. 
+
+- [ ] API release tracker available on wiki
+- [ ] Review comments addressed (by Sub Project)
+- [ ] Release PR approved (on behalf of Release Management)
+- [ ] PR merged (by Sub Project codeowner)
+- [ ] Release created within GitHub (by Sub Project codeowner)
+- [ ] Release Tracker updated (wuth creation date of the release)
```
Comment (hdamker) [inline .github/ISSUE_TEMPLATE/issue_review_template.md:35]:
```suggestion
- [ ] Release Tracker updated (with creation date of the release)
```

File: .github/ISSUE_TEMPLATE/issue_review_template.md [hunk 2]
```diff
@@ -10,18 +10,29 @@ assignees: ''
...
-**Reviewer actions**
+**Review actions**
 
-To review, assign this issue to yourself and follow the below list.
-For any item, review the file(s) in the issue/PR listed above. 
-Put comments in the above issue or PR if they concern non-changed text.
-When done, tick the box in this issue. 
+Assign this issue to yourself and follow the below list.
+For any review action, review the file(s) in the issue/PR listed above. 
+Put comments in the above issue or PR if they concern non-changed files/text.
```
Comment (hdamker) [inline .github/ISSUE_TEMPLATE/issue_review_template.md:17]: I've just done the review in https://github.com/camaraproject/ReleaseManagement/issues/57 and thought that it is good to add a short review summary in addition to ticking the points of: 

```suggestion
Put comments in the above issue or PR if they concern non-changed files/text.
Put a short summary of the main review result here into the review issue.
```

File: .github/ISSUE_TEMPLATE/issue_review_template.md (hunk 1 above)
Comment (tanjadegroot) [reply .github/ISSUE_TEMPLATE/issue_review_template.md]: Have corrected that separately - so done

File: .github/ISSUE_TEMPLATE/issue_review_template.md (hunk 2 above)
Comment (tanjadegroot) [reply .github/ISSUE_TEMPLATE/issue_review_template.md:17]: Maybe a bit double ? the comments are in the PR and in the issue, but ok if short.

**Review by hdamker — APPROVED** [review]

**Pull request merged**